README.md

```diff
@@ -16,7 +16,8 @@ IoT Platform service will also be able to serve the flight information to all th
 ## Architecture
 Following figure shows the high-level architecture of a Cloud-based Air Traffic Control that relies on cheap Ground Stations to track flights
 
-![alt tag](https://github.com/IBM/air-traffic-control/blob/master/assets/architecture_diagram_v2.png)
+![alt tag](https://github.com/ljbennett62/air-traffic-control/blob/master/assets/arch-iot-airtrafficcontrol-1024x878.png)
```

Comment:
The arrows on some of the lines in the diagram need to be switched. Once again, it would be better to schedule a meeting to talk about these things.